src/providers/EtherspotContextProvider.tsx

```diff
@@ -511,6 +512,41 @@ const EtherspotContextProvider = ({
     [sdk, accountAddress]
   );
 
+  // get ENS Node
+  const getENSNode = useCallback(
+    async (
+      chainId: number,
+      address: string | null = accountAddress,
+      recompute: boolean = true
+    ): Promise<ENSNode | null> => {
+      const sdk = getSdkForChainId(chainId);
+
+      if (!sdk) return null;
+      let computedAccount;
+      if (!address && recompute) {
+        try {
+          computedAccount = await connect();
+        } catch (e) {
+          //
+        }
+      }
```

Comment:
Maybe good to add a couple of newlines for readability?
```suggestion
      if (!sdk) return null;
      
      let computedAccount;
      
      if (!address && recompute) {
        try {
          computedAccount = await connect();
        } catch (e) {
          //
        }
      }
```